### PR TITLE
update GSoD page to follow application instructions

### PR DESCRIPTION
**Pull request by mfc**
see instructions: https://developers.google.com/season-of-docs/docs/org-proposal-template

### CODE REVIEW

File: developer/general/gsod.md [hunk 1]
```diff
@@ -8,101 +8,81 @@ permalink: /gsod/
...
+    Propose updated and clarified content
 
- - [Markdown][markdown]
+    Provide a new option of installation guide for users working on virtual machines. 
 
-**Mentor**: [Marek Marczykowski-Górecki][team]
+    Review existing problems and provide solutions to them. 
 
-### Create guide on firstboot for new users
+    Giving a warning for using outdated versions of Qubes.
```
Comment (PROTechThor) [inline developer/general/gsod.md:45]: This part should be rendered as a list, not as code. Let's replace the indents with hyphens.

File: developer/general/gsod.md [hunk 2]
```diff
@@ -8,101 +8,81 @@ permalink: /gsod/
...
+    Basic Qubes OS knowledge - intro, getting started.
 
- - Review [past discussions on the issue](https://github.com/QubesOS/qubes-issues/issues/1774)
- - Provide visual mock-ups and proposed text 
-  
-**Knowledge prerequisite**: 
+    Markdown.
 
- - some experience with Anaconda would be helpful
+    Experience in virtual boxes and machines.
```
Comment (PROTechThor) [inline developer/general/gsod.md:54]: This part too

File: developer/general/gsod.md [hunk 3]
```diff
@@ -8,101 +8,81 @@ permalink: /gsod/
...
-**Expected results**:
+We will consider the project successful if, after publication of the new documentation:
 
-- Review existing Disposable VM documentation
-- Propose new documentation layout, including split between pages
-- Propose updated and clarified content
+    The Disposable VM documentation is consilidated into a few pages and easier to find.
 
-**Knowledge prerequisite**:
+    Disposable VM terminology is consistent across all pages
```
Comment (PROTechThor) [inline developer/general/gsod.md:67]: Same here

File: developer/general/gsod.md [hunk 4]
```diff
@@ -8,101 +8,81 @@ permalink: /gsod/
...
- -Giving a warning for using outdated versions.
-
-**Knowledge prerequisite**: 
- - Experience in virtual boxes and machines.
- - Basic Knowledge about Fedora linux architecture.
- - [Markdown][markdown]
- 
-**Mentor**: [Marek Marczykowski-Górecki][team]
+You can view the project we had in 2019 in the [2019 GSoD archive](https://developers.google.com/season-of-docs/docs/2019/participants/project-qubes) and the [2019 writer’s report](https://refre.ch/report-qubesos/).
+You can also view the project we had in 2020 in the [2020 GSoD archive](https://developers.google.com/season-of-docs/docs/2020/participants/project-qubesos-c1e0) and the [2020 writer’s report](https://gist.github.com/PROTechThor/bfe9b8b28295d88c438b6f6c754ae733).
```
Comment (PROTechThor) [inline developer/general/gsod.md:85]: We can remove this section, since it's already available below under "Past Projects"

**Comment by mfc** [comment]
thanks for catching those, have corrected

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of 9c87e51d9cc2d5f642aa742b98cfd88aea07b1f7
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmBdBxoACgkQ8yiUvpaE
k4p2dRAAsP40s2yNtZCyvMtFw5nlRZI9QYNw0qxKUbmyX5uCBXShJdkYEWM+ywRS
xjOjWZloX3Fj/E1HN8p+nqZ9qmBJZ4guyJafsgVZfbvUN+Bx8Crb/xMKRxA5LaD9
2+cZIV2h8kFP33cllzxf/KKj4s0fLZ0PE5ts221ePCA1h9KqrTV0b8ZIyAjqIew1
a3tbDbp7enQWSXlgDKZZSc3RuysQr7mXlb/f3q73LBa6FG+U90lKpzEwjX40TZto
+v8C51jJWAcFmZBAz6eVxOGIWfaUbGXyogGaM066j5oW1WZY4Z/XgjZ9Kl5sr7D4
JUXkaoc1yIci6X1tzMkJvCv2pWxxLZ9VDWXokYtQKapfaQuIOWLOd/DSK/57DrzM
vD3azGMWRd//ZLUpXEeFmMJYf5PRO2N06o20PiqrNtWy5Cfrg9BNcTWdEAeu4ZN2
qrOv02ob0YjYIE4cGpTJK9vjx0XwOrhJix7JUi7A/g5TlKPR1GwJ+sCJMWcm8Y0S
OFTVEYxUPzo5GK10gdjQe31z3yCJV5C9YsGFwDW8tuh0NPg2+g44utTqjlPRT6fM
2yByt98vyYJq5hvjke3NDyfOvlEPkeTP4Q3xuYmK0xN+hf6eW2NY8nHDyfDlaCPR
k5iYEq53ULIDewJj28DB8PpkXfVKrxHRt/wSiRO/yDXVIbbdFh8=
=eZTi
-----END PGP SIGNATURE-----
```

**Pull request merged**